test/e2e/windows/host_process.go

```diff
@@ -154,8 +188,201 @@ var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] [Excluded:WindowsDoc
 		}
 		framework.ExpectEqual(p.Status.Phase, v1.PodSucceeded)
 	})
+
+	ginkgo.It("should support various volume mount types", func() {
+		ns := f.Namespace
+
+		ginkgo.By("Creating a configmap containing test data and a validation script")
+		configMap := &v1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: "v1",
+				Kind:       "ConfigMap",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "sample-config-map",
+			},
+			Data: map[string]string{
+				"text":              "Lorem ipsum dolor sit amet",
+				"validation-script": validation_script,
+			},
+		}
+		_, err := f.ClientSet.CoreV1().ConfigMaps(ns.Name).Create(context.TODO(), configMap, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "unable to create create configmap")
+
+		ginkgo.By("Creating a secret containing test data")
+		secret := &v1.Secret{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: "v1",
+				Kind:       "Secret",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "sample-secret",
+			},
+			Type: v1.SecretTypeOpaque,
+			Data: map[string][]byte{
+				"foo": []byte("bar"),
+			},
+		}
+		_, err = f.ClientSet.CoreV1().Secrets(ns.Name).Create(context.TODO(), secret, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "unable to create secret")
+
+		ginkgo.By("Creating a pod with a HostProcess container that uses various types of volume mounts")
+
+		podAndContainerName := "host-process-volume-mounts"
+		pod := makeTestPodWithVolumeMounts(podAndContainerName)
+
+		f.PodClient().Create(pod)
+
+		ginkgo.By("Waiting for pod to run")
+		f.PodClient().WaitForFinish(podAndContainerName, 3*time.Minute)
+
+		logs, err := e2epod.GetPodLogs(f.ClientSet, ns.Name, podAndContainerName, podAndContainerName)
+		framework.ExpectNoError(err, "Error getting pod logs")
+		framework.Logf("Container logs: %s", logs)
```

Comment:
I was thinking this should only be done in the case of failure so it would help understand why this test might have failed from reading the test logs.  I don't think the logs would be verbose in the success state so It might be fine here

Reply:
Yea, right now i'll just output SUCCESS which shouldn't bloat the logs too much.